### DISCUSSION TITLE
Hide a list page in the cms

### DISCUSSION THREAD

**Discussion by tocih50254** [decaporg/decap-cms#6908] [Q&A] [answered]
Hi there,

I am using the Decap cms with Hugo, in the collection it show all the pages from directory.

However the client keep messing the list page, how can I hide the list page from the cms

```
# Hugo single page =
index.md

# Hugo List page =
_index.md
```

Any solution will work either page front matter or other cms config setting which can help me achieve this will be great

Thanks

**Comment by martinjagodic** [comment, marked as answer]
This feature requests exists, and also a workaround, which you can see here: https://github.com/decaporg/decap-cms/issues/2727#issuecomment-658469466
